cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -92,7 +96,7 @@ def pytest_generate_tests(metafunc):
                     continue
 
             # Set VM name here
-            vm_name = 'test_ssa_{}-{}'.format(fauxfactory.gen_alphanumeric(), vm_analysis_key)
+            vm_name = 'ssa_{}-{}'.format(fauxfactory.gen_alphanumeric(), vm_analysis_key)
```

Comment:
That doesn't seem to affect anything, sometimes it goes
```
[I] VM ssa_E3zPs24yMj-rhel7_xfs provisioned, waiting for IP address to be assigned (cfme/tests/infrastructure/test_instance_analysis.py:210)
[I] Power state for ssa_E3zPs24yMj-rhel7_xfs vm: powering_up (cfme/tests/infrastructure/test_instance_analysis.py:214)
[I] Fetched IP for ssa_E3zPs24yMj-rhel7_xfs: None (cfme/tests/infrastructure/test_instance_analysis.py:219)
[I] Power state for ssa_E3zPs24yMj-rhel7_xfs vm: up (cfme/tests/infrastructure/test_instance_analysis.py:214)
[I] Fetched IP for ssa_E3zPs24yMj-rhel7_xfs: None (cfme/tests/infrastructure/test_instance_analysis.py:219)
[I] Power state for ssa_E3zPs24yMj-rhel7_xfs vm: up (cfme/tests/infrastructure/test_instance_analysis.py:214)
[I] Fetched IP for ssa_E3zPs24yMj-rhel7_xfs: 10.16.7.87 (cfme/tests/infrastructure/test_instance_analysis.py:219)
[I] Waiting for 10.16.7.87 to be available via SSH (cfme/tests/infrastructure/test_instance_analysis.py:229)
[I] Running command `cat /proc/uptime` (utils/ssh.py:138)
```
and next time the instance is 'fighting' with some script:
```
 Waiting for vm ssa_XGEu5op62P-fedora22-ext4 to appear on provider vsphere55 (cfme/provisioning.py:241)
 Waiting for cfme provision request for vm ssa_XGEu5op62P-fedora22-ext4 (cfme/provisioning.py:245)
 VM ssa_XGEu5op62P-fedora22-ext4 provisioned, waiting for IP address to be assigned (cfme/tests/infrastructure/test_instance_analysis.py:216)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOn (cfme/tests/infrastructure/test_instance_analysis.py:221)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
 Fetched IP for ssa_XGEu5op62P-fedora22-ext4: None (cfme/tests/infrastructure/test_instance_analysis.py:226)
 Power state for ssa_XGEu5op62P-fedora22-ext4 vm: poweredOff (cfme/tests/infrastructure/test_instance_analysis.py:221)
  Starting vSphere VM ssa_XGEu5op62P-fedora22-ext4 (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:330)
  Waiting for vSphere VM ssa_XGEu5op62P-fedora22-ext4 to change status to ON (/usr/lib/python2.7/site-packages/mgmtsystem/virtualcenter.py:420)
```